src/LendingPool.sol

```diff
@@ -249,8 +255,10 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken {
             _deposit(amount, vault);
         }
 
+        uint64 _utilisation = uint64(realisedDebt / totalRealisedLiquidity);
```

Comment:
use a uint256 for variables stored in memory

Reply:
Because it is parsed to uint256 in evm level?

Reply:
yes